src/main/java/io/appform/dropwizard/actors/common/Constants.java

```diff
@@ -0,0 +1,8 @@
+package io.appform.dropwizard.actors.common;
+
+public class Constants {
```

Comment:
Add a private constructor

Reply:
Done. Added for other Util classes as well

Reply:
NoArgsConstructor